src/Message/CreatePaymentMethodRequest.php

```diff
@@ -306,11 +332,16 @@ public function getData($paymentMethodType = self::PAYMENT_METHOD_CREDIT_CARD)
             $this->validate('card');
         }
 
+        $paymentMethod = $this->buildPaymentMethod($paymentMethodType, true);
+        if ($this->getDeactivatePaymentMethod() === true) {
+            $paymentMethod->active = false;
```

Comment:
Rather than setting ```active``` after creating out the object, could you pass it as an option to ```buildPaymentMethod()```?

Reply:
the buildPaymentMethod() is also reused in other requests, and since this is the only place possible to deactivate a payment method, I choose to enclose the changes only to this specific request.